wikitools/category.py

```diff
@@ -1,4 +1,4 @@
-﻿# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*-
```

Comment:
Apparently in December 2008, byte order marks (U+FEFF) were added to files for "utf-8": 87080557d7f4651b88f378765ff0c7ea8cce2a4b.